extensions/pkg/webhook/utils_test.go

```diff
@@ -15,143 +15,963 @@
+				identicalPVC := pvcs[0]
+				result := webhook.EnsurePVCWithName(pvcs, identicalPVC)
+				Expect(result).To(Equal(pvcs))
+			})
+		})
+
+		Describe("#EnsureFileWithPath", func() {
+			var files []extensionsv1alpha1.File
+
+			BeforeEach(func() {
+				files = []extensionsv1alpha1.File{
+					{
+						Path: "/foo.txt",
+						Content: extensionsv1alpha1.FileContent{
+							Inline: &extensionsv1alpha1.FileContentInline{
+								Data: "foo",
+							},
+						},
+					},
+					{
+						Path: "/bar.txt",
+						Content: extensionsv1alpha1.FileContent{
+							Inline: &extensionsv1alpha1.FileContentInline{
+								Data: "bar",
+							},
+						},
+					},
+				}
+			})
+
+			It("should append file when file with such path does not exist", func() {
+				newFile := extensionsv1alpha1.File{
+					Path: "/baz.txt",
 					Content: extensionsv1alpha1.FileContent{
 						Inline: &extensionsv1alpha1.FileContentInline{
 							Data: "baz",
 						},
 					},
-				},
-				{
-					Path: "/bar.txt",
+				}
+
+				actual := webhook.EnsureFileWithPath(files, newFile)
+				Expect(actual).To(Equal(append(files, newFile)))
+			})
+
+			It("should update file when file with such path exists", func() {
+				newFile := extensionsv1alpha1.File{
+					Path: "/foo.txt",
 					Content: extensionsv1alpha1.FileContent{
 						Inline: &extensionsv1alpha1.FileContentInline{
-							Data: "bar",
+							Data: "baz",
 						},
 					},
-				},
-			}))
-		})
-
-		It("should do nothing when the new file is exactly the same as the existing one", func() {
-			newFile := files[0]
+				}
 
-			actual := webhook.EnsureFileWithPath(files, newFile)
-			Expect(actual).To(Equal(files))
-		})
-	})
+				actual := webhook.EnsureFileWithPath(files, newFile)
+				Expect(actual).To(Equal([]extensionsv1alpha1.File{
+					{
+						Path: "/foo.txt",
+						Content: extensionsv1alpha1.FileContent{
+							Inline: &extensionsv1alpha1.FileContentInline{
+								Data: "baz",
+							},
+						},
+					},
+					{
+						Path: "/bar.txt",
+						Content: extensionsv1alpha1.FileContent{
+							Inline: &extensionsv1alpha1.FileContentInline{
+								Data: "bar",
+							},
+						},
+					},
+				}))
+			})
 
-	Describe("#EnsureUnitWithName", func() {
-		var units []extensionsv1alpha1.Unit
+			It("should do nothing when the new file is exactly the same as the existing one", func() {
+				newFile := files[0]
 
-		BeforeEach(func() {
-			units = []extensionsv1alpha1.Unit{
-				{
-					Name:    "foo.service",
-					Content: ptr.To("foo"),
-				},
-				{
-					Name:    "bar.service",
-					Content: ptr.To("bar"),
-				},
-			}
+				actual := webhook.EnsureFileWithPath(files, newFile)
+				Expect(actual).To(Equal(files))
+			})
 		})
 
-		It("should append unit when unit with such name does not exist", func() {
-			newUnit := extensionsv1alpha1.Unit{
-				Name:    "baz.service",
-				Content: ptr.To("bar"),
-			}
+		Describe("#EnsureUnitWithName", func() {
```

Comment:
A lot of `Describe`s are added inside this `Describe`. Looks like something went wrong during the rebase.
Consider fixing by adding `})` at L584 and deleting one `})` from L973, VS code should do the proper formatting then.